### PR TITLE
T-32182 Migrate biswap_bnb_trades

### PR DESCRIPTION
**Pull request by couralex6**

### CODE REVIEW

File: models/biswap/bnb/biswap_bnb_trades.sql [hunk 1]
```diff
@@ -16,13 +16,13 @@
...
+            CASE WHEN amount0Out = UINT256 '0' THEN amount1Out ELSE amount0Out END AS token_bought_amount_raw,
+            CASE WHEN amount0In = UINT256 '0' THEN amount1In ELSE amount0In END    AS token_sold_amount_raw,
             cast(NULL as double)                                         AS amount_usd,
-            CASE WHEN amount0Out = 0 THEN token1 ELSE token0 END         AS token_bought_address,
-            CASE WHEN amount0In = 0 THEN token1 ELSE token0 END          AS token_sold_address,
+            CASE WHEN amount0Out = UINT256 '0' THEN token1 ELSE token0 END         AS token_bought_address,
+            CASE WHEN amount0In = UINT256 '0' THEN token1 ELSE token0 END          AS token_sold_address,
             t.contract_address                                           AS project_contract_address,
             t.evt_tx_hash                                                AS tx_hash,
             ''                                                           AS trace_address,
```
Comment (aalan3) [inline models/biswap/bnb/biswap_bnb_trades.sql:28]: This can be dropped

File: models/biswap/bnb/biswap_bnb_trades.sql [hunk 2]
```diff
@@ -51,31 +51,31 @@ SELECT 'bnb'                                                           AS blockc
...
        biswap_dex.token_sold_address,
-       coalesce(biswap_dex.taker, tx.from)                             AS taker,
+       coalesce(biswap_dex.taker, tx."from")                             AS taker,
        biswap_dex.maker,
        biswap_dex.project_contract_address,
        biswap_dex.tx_hash,
-       tx.from                                                         AS tx_from,
+       tx."from"                                                         AS tx_from,
        tx.to                                                           AS tx_to,
        biswap_dex.trace_address,
```
Comment (aalan3) [inline models/biswap/bnb/biswap_bnb_trades.sql:69]: this too

**Review by aalan3 — APPROVED** [review]
just a lingering column but LGTM. there's another PR that is changing these models here https://github.com/duneanalytics/spellbook/pull/3975

**Comment by chef-seaweed** [comment]
> just a lingering column but LGTM. there's another PR that is changing these models here #3975

Sorry for duplicated works. The Biswap part of my PR mainly focus on 

1. rename v1 to v2
2. change `biswap_bnb_trades`  to meta-table spell 

Not sure how do we settle the difference between PRs here 🙏

**Comment by couralex6** [comment]
> > just a lingering column but LGTM. there's another PR that is changing these models here #3975
> 
> Sorry for duplicated works. The Biswap part of my PR mainly focuses on
> 
> 1. rename v1 to v2 (previous `biswap_bnb_trades` is now `biswap_v2_bnb_trades`)
> 2. change `biswap_bnb_trades`  to meta-table spell
> 
> Not sure how do we settle the difference between PRs here 🙏

I'll close this one! This PR is part of a semi-automated mass migration effort from an internal fork, so it's not a big deal.

**Pull request closed without merging**